superchain/configs/mainnet/lyra.yaml

```diff
@@ -0,0 +1,17 @@
+name: Lyra
+chain_id: 957
+public_rpc: https://rpc.lyra.finance
+sequencer_rpc: https://rpc.lyra.finance
+explorer: https://explorer.lyra.finance
+
+system_config_addr: "0x0e4c4cdd01cecb01070e9fdfe7600871e4ae996e"
+batch_inbox_addr: "0x5f7f7f6db967f0ef10bda0678964dba185d16c50"
+
+genesis:  
+  l1:  # L1 anchor-point of the rollup
+    hash: "0x00b06b23108483a0b6af8ff726b5ed3f508b7986f72c12679b10d72c05839716"
+    number: 18574841
+  l2:
+    hash: "0x047f535b3da7ad4f96d353b5a439740b7591413daee0e2f27dd3aabb24581af2"
+    number: 0
+  l2_time: 1700021615
```

Comment:
This timestamp does not match what is in the uploaded genesis.json for lyra. It does match the genesis.json file you provided me.